### PR TITLE
feat: code obfuscator

### PR DESCRIPTION
**Pull request by zetxx**

### CODE REVIEW

File: lib/obfuscator.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                    fs.writeFileSync(
+                        fn,
+                        JavaScriptObfuscator.obfuscate(fs.readFileSync(fn).toString()).getObfuscatedCode()
+                    );
+                });
+        }
+    }
+};
+
+module.exports.obfuscate();
```
Comment (kalinkrustev) [inline lib/obfuscator.js]: is this line needed?

File: lib/obfuscator.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const JavaScriptObfuscator = require('javascript-obfuscator');
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+const cwd = process.cwd();
+
+// checks for file named "obfuscate" with new line separated records, each record is a glob pattern for matching js files which should be obfuscated
+module.exports = {
+    obfuscate: function(done) {
```
Comment (kalinkrustev) [inline lib/obfuscator.js]: done is not used

File: lib/obfuscator.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        if (fs.existsSync(path.join(cwd, './obfuscate'))) { // check for file
```
Comment (kalinkrustev) [inline lib/obfuscator.js]: maybe name the file .obfuscate, as it appears to be configuration as other dot files

File: lib/obfuscator.js (hunk 1 above)
Comment (zetxx) [inline lib/obfuscator.js]: it looks cooler

**Pull request merged**